### PR TITLE
chromium: update to 88.0.4324.96

### LINKED ISSUE #452: chromium: update to 88.0.4324.96
Stable release announcements since our latest update:
https://chromereleases.googleblog.com/2021/01/stable-channel-update-for-desktop_19.html

Targets and platforms we need to check:

**chromium-x11:**
- dunfell, clang, MACHINE="intel-core2-32"
- dunfell, clang, MACHINE="intel-corei7-64"
- dunfell, clang, MACHINE="raspberrypi3"
- gatesgarth, MACHINE="raspberrypi3"

**chromium-ozone-wayland:**

- dunfell, clang, MACHINE="intel-corei7-64"
- dunfell, clang, MACHINE="qemuarm64"
- gatesgarth, clang, MACHINE="raspberrypi3"
- dunfell, clang, MACHINE="m3ulcb"

### PR DESCRIPTION
**Pull request by msisov**
Stable release announcements since our latest update:
https://chromereleases.googleblog.com/2021/01/stable-channel-update-for-desktop_19.html

Changes:
  * It's actually not required to set proprietary-codecs and it's up to
the user if they want to have them or not.
  * VA-API is compiled by default since M88. Thus, I had to disable that
to avoid pulling libva dependency for chromium x11 and wayland recipes
if va-api is not actually needed.

Test-built with:

* chromium-x11
  * dunfell, clang, MACHINE="intel-core2-32"
  * dunfell, clang, MACHINE="intel-corei7-64"
  * dunfell, clang, MACHINE="raspberrypi3"
  * gatesgarth, MACHINE="raspberrypi3"

* chromium-ozone-wayland:
  * dunfell, clang, MACHINE="intel-corei7-64"
  * dunfell, clang, MACHINE="qemuarm64"
  * gatesgarth, clang, MACHINE="raspberrypi3"
  * dunfell, clang, MACHINE="m3ulcb"

License Changes
---------------
Updated licenses:
  * third_party/android_deps/libs/com_google_android_gms_play_services_auth_api_phone/LICENSE
  - [WebOTP] Update gms_play_services_auth_api_phone to 17.5.0
  - https://chromium-review.googlesource.com/c/chromium/src/+/2485192

  * third_party/android_deps/libs/com_google_android_gms_play_services_base/LICENSE
  - [WebOTP] Update gms_play_services_auth_api_phone to 17.5.0
  - https://chromium-review.googlesource.com/c/chromium/src/+/2485192

  * third_party/android_deps/libs/com_google_android_gms_play_services_basement/LICENSE
  - [WebOTP] Update gms_play_services_auth_api_phone to 17.5.0
  - https://chromium-review.googlesource.com/c/chromium/src/+/2485192

  * third_party/icu/LICENSE
  - Roll ICU to fix ListFormat security bug
  - https://chromium-review.googlesource.com/c/chromium/src/+/2555661

  * third_party/woff2/LICENSE
  - Roll WOFF2 1bccf208bc..a0d0ed7da2
  - https://chromium-review.googlesource.com/c/chromium/src/+/2465555

Removed licenses:
  * chrome/android/third_party/compositor_animator/LICENSE
  - Migrate CompositorAnimator code to layouts module
  - https://chromium-review.googlesource.com/c/chromium/src/+/2503830

  * third_party/libovr/LICENSE
  - Remove Oculus VR Runtime and SDK/Lib
  - https://chromium-review.googlesource.com/c/chromium/src/+/2435722

Added licenses:
  * chrome/browser/ui/android/layouts/third_party/float_property/LICENSE
  - Migrate CompositorAnimator code to layouts module
  - https://chromium-review.googlesource.com/c/chromium/src/+/2503830

  * third_party/android_deps/libs/androidx_core_core_animation/LICENSE
  - Add androidx_core_animation.
  - https://chromium-review.googlesource.com/c/chromium/src/+/2528122

  * third_party/android_deps/libs/com_android_support_animated_vector_drawable/LICENSE
  - Android: Use android_deps for play services APIs
  - https://chromium-review.googlesource.com/1142189

  * third_party/android_deps/libs/com_android_support_appcompat_v7/LICENSE
  - Android: Use android_deps for play services APIs
  - https://chromium-review.googlesource.com/1142189

  * third_party/android_deps/libs/com_android_support_cardview_v7/LICENSE
  - Android: Use android_deps for play services APIs
  - https://chromium-review.googlesource.com/1142189

  * third_party/android_deps/libs/com_android_support_design/LICENSE
  - Android: Use android_deps for play services APIs
  - https://chromium-review.googlesource.com/1142189

  * third_party/android_deps/libs/com_android_support_recyclerview_v7/LICENSE
  - Android: Use android_deps for play services APIs
  - https://chromium-review.googlesource.com/1142189

  * third_party/android_deps/libs/com_android_support_support_vector_drawable/LICENSE
  - Android: Use android_deps for play services APIs
  - https://chromium-review.googlesource.com/1142189

  * third_party/android_deps/libs/com_android_support_transition/LICENSE
  - Android: Use android_deps for play services APIs
  - https://chromium-review.googlesource.com/1142189

  * third_party/android_deps/libs/com_google_errorprone_error_prone_annotations/LICENSE
  - Android: Add missing dependencies in android_deps
  - https://chromium-review.googlesource.com/1208462

  * third_party/android_deps/libs/org_checkerframework_checker_compat_qual/LICENSE
  - [Build] Update checkerframework licenses
  - https://chromium-review.googlesource.com/c/chromium/src/+/2646844

  * third_party/android_deps/libs/org_codehaus_mojo_animal_sniffer_annotations/LICENSE
  - Android: Add missing dependencies in android_deps
  - https://chromium-review.googlesource.com/1208462

  * third_party/liburlpattern/LICENSE
  - Add third_party/liburlpattern directory.
  - https://chromium-review.googlesource.com/c/chromium/src/+/2486388

  * third_party/libx11/LICENSE
  - Add subsets of libxcb-keysyms, libx11, and x11proto to third_party
  - https://chromium-review.googlesource.com/c/chromium/src/+/2500268

  * third_party/libxcb-keysyms/LICENSE
  - Add subsets of libxcb-keysyms, libx11, and x11proto to third_party
  - https://chromium-review.googlesource.com/c/chromium/src/+/2500268

  * third_party/subresource-filter-ruleset/LICENSE
  - Move subresource filter ruleset for perf tests to Google storage
  - https://chromium-review.googlesource.com/c/chromium/src/+/1959652

  * third_party/x11proto/LICENSE
  - Add subsets of libxcb-keysyms, libx11, and x11proto to third_party
  - https://chromium-review.googlesource.com/c/chromium/src/+/2500268

Fixes #452

### CODE REVIEW

File: recipes-browser/chromium/chromium-gn.inc [hunk 1]
```diff
@@ -117,6 +117,13 @@ PACKAGECONFIG[proprietary-codecs] = ' \
         ffmpeg_branding="Chromium" proprietary_codecs=false \
 '
 
+# Disable VA-API by default. It is compile time enabled since M88, but it's not
```
Comment (rakuco) [inline recipes-browser/chromium/chromium-gn.inc:120]: Can you update the option's description in the README?

**Review by rakuco — APPROVED** [review]

File: recipes-browser/chromium/chromium-gn.inc (hunk 1 above)
Comment (msisov) [reply recipes-browser/chromium/chromium-gn.inc:120]: done.

**Review by rakuco — APPROVED** [review]
lgtm!

**Comment by msisov** [comment]
> lgtm!

thanks!

**Pull request merged**